piper/leddialog.py

```diff
@@ -60,6 +61,11 @@ def __init__(self, ratbagd_led, *args, **kwargs):
         self.adjustment_brightness.set_value(self._led.brightness)
         self.adjustment_effect_rate.set_value(self._led.effect_rate)
 
+        sp = Gtk.CssProvider()
+        sp.load_from_data("* { background: #565854}".encode())
```

Comment:
In general I don't like hardcoding colors because this may break completely with different themes. Isn't there a color we can retrieve from the theme (through https://developer.gnome.org/gtk3/stable/GtkStyleContext.html#gtk-style-context-get-color) such that it will work for all themes? This is the approach used in the mousemap as well.

Reply:
looks like that function is deprecated, so we shouldn't use it. it directs me to [gtk_render_background](https://developer.gnome.org/gtk3/stable/GtkStyleContext.html#gtk-render-background) instead, but that suggests its based on CSS data anyway? That magic value is taken from the gnome-control-center, so that too would break with nondefault themes.

Reply:
> looks like that function is deprecated

Where do you see that?

> it directs me to `gtk_render_background` instead, but that suggests its based on CSS data anyway?

CSS data taken from whatever theme the user has applied, yes. Not a magic value taken from gnome-control-center that only works with Adwaita. At least, if I understand it correctly.

Reply:
>> looks like that function is deprecated
>Where do you see that?

Oh, oops. I just see I've been looking at the foreground color version and not the background color version, which is indeed deprecated. `gtk_render_background` seems to be the wrong function as I think it has to be used in the `draw` function only.

In that case, let's go with this and revisit if people start complaining about custom themes breaking.